lnrpc/rpc.proto

```diff
@@ -3022,14 +3029,46 @@ message ListPaymentsRequest {
     /**
     If true, then return payments that have not yet fully completed. This means
     that pending payments, as well as failed payments will show up if this
-    field is set to True.
+    field is set to true. This flag doesn't change the meaning of the indices,
+    which are tied to individual payments.
+    */
+    bool include_incomplete = 1 [json_name = "include_incomplete"];
```

Comment:
Since this PR was opened, we've remove json from `rpc.proto`. They're now just set to the field's name, so you can remove `json_name`...

Reply:
Missed that, good to know!